scripts/auth-common.sh

```diff
@@ -13,16 +13,8 @@ fi
 
 set +x
 IRONIC_HTPASSWD_FILE=/etc/ironic/htpasswd
-if [[ -f "/auth/ironic/username" ]]; then
-    IRONIC_HTPASSWD_USERNAME=$(</auth/ironic/username)
-fi
-IRONIC_HTPASSWD_USERNAME=${IRONIC_HTPASSWD_USERNAME:-}
-if [[ -f "/auth/ironic/password" ]]; then
-    IRONIC_HTPASSWD_PASSWORD=$(</auth/ironic/password)
-fi
-IRONIC_HTPASSWD_PASSWORD=${IRONIC_HTPASSWD_PASSWORD:-}
-if [[ -n "${IRONIC_HTPASSWD_USERNAME}" ]]; then
-    IRONIC_HTPASSWD="$(htpasswd -n -b -B "${IRONIC_HTPASSWD_USERNAME}" "${IRONIC_HTPASSWD_PASSWORD}")"
+if [[ -f "/auth/ironic/htpasswd" ]]; then
+    IRONIC_HTPASSWD=$(</auth/ironic/htpasswd)
 fi
 export IRONIC_HTPASSWD=${IRONIC_HTPASSWD:-${HTTP_BASIC_HTPASSWD:-}}
 set -x
```

Comment:
Same here.

Reply:
Yeah, I don't see a huge problem in logging the htpasswd string.

Reply:
But is this a problem? I mean I get that it is an overkill but I don't think it is computationally expensive so can't we just leave it like that ? I would like to leave it as it is.

Reply:
It might make debugging harder as you miss part of the flow without any logging.

Reply:
this should probably be changed now, CI needs to rerun anyway

Reply:
Removed.